words.yml

```diff
@@ -1,10 +0,0 @@
-- words:
```

Comment:
We'll probably end up some uncategorized words, but we could wait until we get there to see how that looks. I like the idea of categories as you've done them so far :)